pc_ble_driver_py/ble_driver.py

```diff
@@ -2279,6 +2279,26 @@ def ble_gap_adv_data_set(self, adv_data=BLEAdvData(), scan_data=BLEAdvData()):
         return driver.sd_ble_gap_adv_data_set(
             self.rpc_adapter, p_adv_data, adv_data_len, p_scan_data, scan_data_len
         )
+        
+    @NordicSemiErrorCheck
+    @wrapt.synchronized(api_lock)
+    def ble_gap_device_name_set(self, name, device_name_read_only=True):
+        write_perm=BLEGapConnSecMode()
+        if device_name_read_only:
+            write_perm.sm = 0
+            write_perm.lv = 0
+        else:
+            write_perm.sm = 1
+            write_perm.lv = 1
```

Comment:
write_perm.set_open()

Reply:
Ok! Now using the BLEGapConnSecMode methods for setting write permissions for the name.

Also trying to write a test for ble_gap_device_name_set()